backend/provisioner/scaling/k8sscaling/k8s_scaling.go

```diff
@@ -394,6 +394,32 @@ func (r *k8sScaling) handleNewDeployment(ctx context.Context, realm string, modu
 		}
 	}
 
+	// Create a ConfigMap for the secrets and config
```

Comment:
I feel like this should be a separate config/secrets provisioner, and not part of k8s scaling? I guess this will be moot once we move to Helm.